libraries/standard/http/cbmc/proofs/HTTPClient_Send/Makefile

```diff
@@ -0,0 +1,34 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HARNESS_ENTRY=harness
+HARNESS_FILE=HTTPClient_Send_harness
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY += __CPROVER_file_local_http_client_c_findHeaderFieldParserCallback
+REMOVE_FUNCTION_BODY += __CPROVER_file_local_http_client_c_findHeaderValueParserCallback
```

Comment:
findHeaderValueParserCallback and findHeaderFiledParserCallback code does not coincide with HTTPClient_Send. Could you please delete this copy and paste.

Reply:
I think it is better to remove these functions so that function pointer targets aren't replaced with them in transport interface send/recv, leading to faster execution time.

Reply:
Oh that would be a great comment in this file for others who are checking it out to learn about CBMC